VoodooI2CHID/VoodooI2CHIDDevice.cpp

```diff
@@ -290,10 +238,10 @@ void VoodooI2CHIDDevice::releaseResources() {
         interrupt_simulator = NULL;
     }
 
-    if (interrupt_source) {
-        interrupt_source->release();
-        interrupt_source = NULL;
-    }
+
+    api->disableInterrupt(0);
```

Comment:
You should  call stopInterrupts as @Goshin has address this in https://github.com/VoodooI2C/VoodooI2CHID/commit/b52b1b8e416994ba85ec70aa1b901942d3a865c6 and then remove the interrupt source. Why are you removing the `release`??? This will result in a leak as VoodooGPIO will still be held onto.